R/redux.R

```diff
@@ -34,7 +34,7 @@ redisGetContext <- function()
 
 uncerealize <- function(x)
 {
-  if(!is.null(x) && is.raw(x)) unserialize(x) else NULL
+  if(!is.null(x) && is.raw(x)) unserialize(x) else x
```

Comment:
Actually, why not just use `if(is.raw(x)) unserialize(x) else x` here since `NULL` is not raw?

Reply:
You're right. But I'll merge the above which also works and is easy to just merge.